content/en/blog/_posts/2023-02-06-k8s-gcr-io-freeze-announcement.md

```diff
@@ -0,0 +1,36 @@
+---
+layout: blog
+title: "k8s.gcr.io image registry will be frozen from the 3rd of April 2023"
+date: 2023-02-06
+slug: k8s-gcr-io-freeze-announcement
+---
+
+**Authors**: Mahamed Ali (Rackspace Technology)
+
+The Kubernetes project has released a new community-owned image registry called `registry.k8s.io`. This registry has been generally available(GA) for several months, and we have published a [blog post](https://kubernetes.io/blog/2022/11/28/registry-k8s-io-faster-cheaper-ga/) about it, its benefits to the community, and the Kubernetes project. This post also announced that future versions of Kubernetes will not be available in the old registry.
+
+On the 3rd of April 2023, the `k8s.gcr.io` image registry will be frozen and no further images for Kubernetes and related subprojects will be pushed to the old registry.
+
+What does this change mean for contributors:
+- If you are a maintainer of a subproject, you will need to update your manifests and Helm charts to use the new registry.
+
+What does this change mean for end users:
+- 1.27 Kubernetes release will not be published to the old registry.
+- Starting in 1.25, the default image registry has been set to `registry.k8s.io`. This value is overridable but will no longer work for any version of Kubernetes released after April as they won’t be present in the old registry.
+- If you are running Kubernetes in networks where outbound traffic is restricted, you should consider hosting local image registry mirrors.
```

Comment:
```suggestion
- If you want to increase reliability of your cluster and remove dependency on community-owned registry or you are running Kubernetes in networks where external traffic is restricted, you should consider hosting local image registry mirrors. Some cloud vendors may offer hosted solutions for this.
```

Reply:
let's emphasize that reliability will increase by removing dependency on community owned registry

Reply:
ack, copied over to the email doc.